src/web-rtc-client.js

```diff
@@ -20,7 +20,7 @@ import { C } from 'sip.js/lib/core/messages/methods/constants';
 import { URI } from 'sip.js/lib/grammar/uri';
 import { Parser } from 'sip.js/lib/core/messages/parser';
 import { UserAgent } from 'sip.js/lib/api/user-agent';
-import { stripVideo } from 'sip.js/lib/platform/web/modifiers/modifiers';
+import { stripVideo, holdModifier } from 'sip.js/lib/platform/web/modifiers/modifiers';
```

Comment:
Seems this PR have also fixes for hold. Can you make another PR or change the PR description?

Reply:
Done, thx